### DISCUSSION TITLE
Unable to resolve: [read udp 127.0.0.1:59476->127.0.0.1:53: read: connection refused

### DISCUSSION THREAD

**Discussion by KAGEYAM4** [DNSCrypt/dnscrypt-proxy#2494] [Q&A]
```
[phoenix@ArchLinux dnscrypt-proxy]$ dnscrypt-proxy -resolve https://aniwave.to/home
Resolving [https://aniwave.to/home] using 127.0.0.1 port 53

Unable to resolve: [read udp 127.0.0.1:59476->127.0.0.1:53: read: connection refused]
```

After performing https://github.com/DNSCrypt/dnscrypt-proxy/wiki/Installation-linux#step-4-change-the-system-dns-settings
the -resolve command dosen't work.

Also, i am college/university network. Could this be the issue ? How to solve it ?

**Comment by lifenjoiner** [comment]
I would suggest:
1. Make sure the `dnscrypt-proxy` server process is running (step 3), by some process manager app. I guess not.
2. After the previous step is confirmed, try `dnscrypt-proxy -resolve aniwave.to`, rather than the whole URL.

**Reply by KAGEYAM4** [reply]
I refollowed everystep alongwith archwiki steps and now things are working.

By the way can `dnsmasq` create any issue with `dnscrypt`?
![image](https://github.com/DNSCrypt/dnscrypt-proxy/assets/75798544/29995ea2-cf33-4812-b64c-4afa59779929)
